modules/datastore/src/Drush.php

```diff
@@ -45,7 +45,7 @@ public function import($uuid, $deferred = FALSE) {
 
     try {
       // Load metadata with both identifier and data for this request.
-      drupal_static('metastore_dereference_method', Dereferencer::DEREFERENCE_OUTPUT_REFERENCE_IDS);
+      drupal_static('metastore_dereference_method', ResourceMapper::DEREFERENCE_YES);
```

Comment:
@thierrydallacroce I believe we can simply remove these calls to drupal_static. Dereferencing effectively always happens.

Reply:
Done in de7f9c0.